lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -54,11 +52,28 @@ class ProgressTabView(RetrieveAPIView):
                 children: (list) If the block has child blocks, a list of IDs of
                     the child blocks.
         enrollment_mode: (str) a str representing the enrollment the user has ('audit', 'verified', ...)
+        courseware_summary: List of serialized Chapters. each Chapter has the following fields:
+            display_name: (str) a str of what the name of the Chapter is for displaying on the site
+            subsections: List of serialized Subsections, each has the following fields:
+                display_name: (str) a str of what the name of the Subsection is for displaying on the site
+                due: (str) a DateTime string for when the Subsection is due
+                format: (str) the format, if any, of the Subsection (Homework, Exam, etc)
+                graded: (bool) whether or not the Subsection is graded
+                graded_total: an object containing the following fields
+                    earned: (float) the amount of points the user earned
+                    possible: (float) the amount of points the user could have earned
+                percent_graded: (float) the percentage of the points the user received for the subsection
+                show_correctness: (str) a str representing whether to show the problem/practice scores based on due date
+                show_grades: (bool) a bool for whether to show grades based on the access the user has
+                url: (str) the absolute path url to the Subsection
+        user_timezone: (str) The user's preferred timezone
+
+
 
     **Returns**
 
         * 200 on success with above fields.
-        * 403 if the user is not authenticated.
```

Comment:
So I don't think you needed to remove this. You have a test that returns a 403 so this endpoint certainly can do that. I think you want both 403 and 302 documented.